src/components/ObsDetails/ActivityItem.js

```diff
@@ -95,20 +96,27 @@ const ActivityItem = ( {
             )}
           {item.body && currentUser
             ? (
-              <KebabMenu>
+              <KebabMenu
+                visible={kebabMenuVisible}
+                setVisible={setKebabMenuVisible}
```

Comment:
I can see why you're doing this and I don't think this needs to block, but this complicates the use of this component, in that every context that uses it now needs to control the state of whether the menu is open or not. I'm just not sure what a better approach would be. Best I can come up with is to have KebabMenu not accept children and instead accept a prop that's an array of Menu.Item props, and then KebabMenu wraps the onPress prop of each to execute the function and then set the internal visible state to false. Maybe a bit complicated.